### DISCUSSION TITLE
外网访问OpenWRT: Web管理

### DISCUSSION THREAD

**Discussion by josenzhong** [coolsnowwolf/lede#9663] [Bugs / 问题反馈]
我设置了Openwrt的Web管理页面x掉了“只允许内网访问”，并把端口转发设置好了，有公网IP，还是没法从外网打开？

我连在软路由上的NAS做好端口转发可以正常外网访问。

是不是“只允许内网访问”这一块固件里有Bug? Lean大能帮忙看看吗？如果可以麻烦在下一版的固件里修复一下，谢谢！

界面截图：
![Screenshot_20220628-204606_Chrome.jpg](https://user-images.githubusercontent.com/11269628/176183805-0980c1cc-5d35-467c-9847-537cdc80a1c5.jpg)

**Comment by ZaberKo** [comment]
我感觉这个选项好像没用, 可以在“防火墙”-"通讯规则"设置允许wan上设备访问80或443端口

**Reply by wzsdch** [reply]
非常感谢，解决了我一个困扰了几天的问题

**Comment by ilsjf** [comment]
换个端口号重新设置转发试试

**Comment by frank-pv** [comment]
国内运营商默认禁用80、443、8080、445端口
